cmd/client/cmd/upload.go

```diff
@@ -100,3 +112,35 @@ func newUploadCmd() *cobra.Command {
 
 	return uploadCmd
 }
+
+// readFileFromURL reads translation file from URL.
+func readFileFromURL(ctx context.Context, filePath string) ([]byte, error) {
+	u, err := url.ParseRequestURI(filePath)
+	if err != nil {
+		return nil, fmt.Errorf("parse URL: %w", err)
```

Comment:
Your intent is not to parse, but to validate

```suggestion
		return nil, fmt.Errorf("validate file URL: %w", err)
```